api/institutions/serializers.py

```diff
@@ -48,9 +48,20 @@ def get_api_url(self, obj):
     def get_absolute_url(self, obj):
         return obj.absolute_api_v2_url
 
+    def get_assets(self, obj):
+        return {
+            'logo': obj.logo_path,
+            'logo_rounded': obj.logo_path_rounded_corners,
+        }
+
     class Meta:
         type_ = 'institutions'
 
+    # Deprecated fields
+    logo_path = ShowIfVersion(
```

Comment:
You need to import `ShowIfVersion`, [e.g.](https://github.com/CenterForOpenScience/osf.io/blob/705769937549264cbdbb4ceca06d4ea7a4e1ee61/api/preprint_providers/serializers.py#L3)